include/RemotePlugin.h

```diff
@@ -141,6 +143,11 @@ class LMMS_EXPORT RemotePlugin : public QObject, public RemotePluginBase
 		m_commMutex.unlock();
 	}
 
+	PluginPinConnector& pinConnector()
+	{
+		return m_pinConnector;
```

Comment:
Also, I would have expected this rather in `Plugin`, since all plugins will probably need to use one?

Reply:
True, I'll see if I can move the pin connector to `Plugin`. I think it should be fine there even if not all plugins use it yet.

Reply:
Right now, I wonder if even `SampleTrack` (which does not use `Plugin`) should have the connector, too? This might even make us put the connector into the `Track` class, though this might very well be overcomplication - we could just add another connector into `SampleTrack`...

Reply:
Moving it to `Track` wouldn't work for effect plugins, so I think just adding a connector into `SampleTrack` would be the way to go. Maybe that should wait until a follow-up PR to make reviewing this one easier.